project-base/app/src/Controller/Admin/TestController.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Admin;
+
+use App\Model\Order\Order;
+use Shopsys\AdministrationBundle\Component\Attributes\CrudController;
+use Shopsys\AdministrationBundle\Component\Config\Action\ActionsConfig;
+use Shopsys\AdministrationBundle\Component\Config\Action\Builder\AbstractAction;
+use Shopsys\AdministrationBundle\Component\Config\Action\Builder\Action;
+use Shopsys\AdministrationBundle\Component\Config\ActionType;
+use Shopsys\AdministrationBundle\Component\Config\CrudConfig;
+use Shopsys\AdministrationBundle\Component\Datagrid\Datagrid;
+use Shopsys\AdministrationBundle\Controller\AbstractCrudController;
+use Shopsys\FrameworkBundle\Model\Pricing\Currency\Currency;
+
+#[CrudController(Order::class)]
+class TestController extends AbstractCrudController
+{
+    /**
+     * @param \Shopsys\AdministrationBundle\Component\Config\CrudConfig $config
+     * @return \Shopsys\AdministrationBundle\Component\Config\CrudConfig
+     */
+    protected function configure(CrudConfig $config): CrudConfig
+    {
+        return $config
+            //->setTitle(PageType::LIST, 'Test listing')
+            //->setMenuTitle(t('Test menu'))
+            //->hideInMenu()
+            //->setActions([PageType::LIST, PageType::EDIT])
+            //->setRoutePrefix('/SomePrefix/prefix-prefixes/hm/')
+            ->setMenuSection('customers', 'promo_codes')
+            ->disableAction(ActionType::CREATE)
+        ;
+    }
+
+    /**
+     * @param \Shopsys\AdministrationBundle\Component\Config\Action\ActionsConfig $actions
+     * @return \Shopsys\AdministrationBundle\Component\Config\Action\ActionsConfig
+     */
+    protected function configureActions(ActionsConfig $actions): ActionsConfig
+    {
+        $actions->add(
+            ActionType::LIST,
+            Action::create('linkToDashboard', t('Link To dashboard'))
+                ->setCssClass('btn--primary')
+                ->setOpenInNewTab()
+                ->linkToRoute('admin_default_dashboard', fn () => [
+                    'id' => 1,
```

Comment:
this does not make sense here but I suppose it does not matter :slightly_smiling_face:

Reply:
It does make sense as I wanted to check if passed parameters are working as expected :D